### PR TITLE
added ecosystem fluxes in the Readme file

### PR DESCRIPTION
**Pull request by jogaudard**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -205,7 +205,26 @@ The biomass from the clipping treatments will be treated in the same way.
...
+**Ecosystem fluxes**
+
+Carbon fluxes are measured with a closed loop chamber system connected to an infrared gaz analyser (INGRA; Li-840A, Li-cor).
+The plexiglas chamber (25x25x40 cm) is connected to the INGRA with plastic tubes (inlet and outlet, 3m long and 4mm diameter each).
+A filter prevents water droplets and small particules from entering the INGRA and an air pump ensures a flow or 1L/mn.
+The chamber is equiped with a fan to mix the air, a thermocouple (Pt1000, Delta-T) to measure air temperature and a PAR sensor.
+Airtightness is ensured by laying a heavy chain on the "skirt" of the chamber.
+The CO2 concentration was measured every seconds and logged in a datalogger (Squirrel 2010).
+The chamber was put on the plot during two minutes for each measurements and then aired during one minute.
+Since the logger was logging continuesly, it was necessary to keep track at what time each measurements started.
```
Comment (audhalbritter) [inline README.Rmd]: ..., the start and end time of each measurement was noted.

File: R/C-flux/description.R [hunk 2]
```diff
@@ -6,22 +6,123 @@ get_file(node = "pk4bg",
...
 
-flux.colnames <- read_csv("Three-D_c-flux_2020.csv") %>% 
+flux <- read_csv("Three-D_c-flux_2020.csv")
+
+flux.colnames <- flux %>% 
   colnames()
 
-variables <- tibble(flux.colnames, description = c(
+variables <- tibble(flux.colnames, "Variable type" = c(
+  # Datetime
```
Comment (audhalbritter) [inline R/C-flux/description.R]: can you do this automatically, by using class()?

File: R/C-flux/description.R [hunk 3]
```diff
@@ -6,22 +6,123 @@ get_file(node = "pk4bg",
...
+  # Campaign
   ""
+),
+"Units/formats/treatment level coding" = c(
+  # Datetime
+  "AC CET",
   # ID
+  paste(range(flux$ID), collapse = " - "),
   # Turf_ID
+  "??", #I am not sure what it is
```
Comment (audhalbritter) [inline R/C-flux/description.R]: it does not have units, but each turf is a unique level. maybe just give one example

File: README.Rmd [hunk 4]
```diff
@@ -240,18 +248,25 @@ Where:
...
-Cleaning and light response curves will be added later (what about pictures?)
+<!-- Fluxes with a non significant slope (>0.05) were removed. -->
+
 <!-- cleaning -->
+<!-- data description -->
+<!-- Make a new table in an R script and call it here. Use the col names from the flux data set and take exemple on the China data paper (https://www.nature.com/articles/s41597-020-0529-0/tables/4?proof=t) -->
+ ```{r, include=FALSE}
+ # source("R/C-Flux/description.R", local = knitr::knit_global())
+ sys.source("description.R", envir = knitr::knit_global())
+ ```
```
Comment (audhalbritter) [inline README.Rmd:263]: I think I would rather source the description file at the top of the rmarkdown file, and mayb call it data_dictionary.R. Then I can add more of these tables. And then here just print the table

Does the table look nice when you knit it?

File: README.Rmd (hunk 1 above)
Comment (jogaudard) [reply README.Rmd]: corrected

File: R/C-flux/description.R (hunk 2 above)
Comment (jogaudard) [reply R/C-flux/description.R]: See my message on slack

File: R/C-flux/description.R (hunk 3 above)
Comment (jogaudard) [reply R/C-flux/description.R]: done

File: README.Rmd (hunk 4 above)
Comment (jogaudard) [reply README.Rmd]: Done.
Yes the table is ok

**Pull request closed without merging**